sensors/nxt_i2c_sensor_defs.c

```diff
@@ -446,6 +476,61 @@ const struct nxt_i2c_sensor_info nxt_i2c_sensor_defs[] = {
 			},
 		},
 	},
+	[LEGO_NXT_TEMPERATURE_SENSOR] = {
+		/**
+		 * @vendor_name: LEGO
+		 * @vendor_part_number: 9749
+		 * @vendor_part_name: MINDSTORMS Temperature Sensor
+		 * @vendor_website: https://education.lego.com/en-us/products/mindstorms-temperature-sensor/9749
+		 * @default_address: 0x4C
+		 */
+		.name		= LEGO_NXT_TEMPERATURE_SENSOR_NAME,
+		.vendor_id	= "LEGO",
+		.product_id	= "", /* LMS2012 fakes this with "Temp." */
```

Comment:
I think in that case, it actually needs to be empty for proper matching. I don't have one for testing though.